src/plugins/discover/public/application/components/default_discover_table/table_header_column.tsx

```diff
@@ -168,7 +144,7 @@ export function TableHeaderColumn({
         values: { columnName: name },
       }),
       className: 'fa fa-angle-double-left osdDocTableHeader__move',
-      onClick: () => onReorderColumn && onReorderColumn(name, currentIdx, colLeftIdx),
+      onClick: () => onReorderColumn && onReorderColumn(name, colLeftIdx),
```

Comment:
If we have this, why are we making a new onMove function?

Reply:
you are right. will remove onMove.